udata/core/dataset/rdf.py

```diff
@@ -119,7 +147,7 @@ def temporal_to_rdf(
 def frequency_to_rdf(frequency: str, graph: Optional[Graph] = None) -> Optional[str]:
     if not frequency:
         return
-    return RDF_FREQUENCIES.get(frequency, getattr(FREQ, frequency))
+    return FREQ_UDATA_TO_TERM.get(frequency) or EUFREQ_UDATA_TO_TERM.get(frequency) or frequency
```

Comment:
This isn't ideal since e.g. `frequency_to_rdf(frequency_from_rdf(EUFREQ.DAILY))` will return `FREQ.daily`.

But short of keeping track of original URIs this is the best we can do.